server/src/test/java/org/apache/druid/server/http/MetadataResourceTest.java

```diff
@@ -129,6 +129,15 @@ public void setUp()
            .when(storageCoordinator)
            .retrieveSegmentForId(segments[5].getId().toString(), true);
 
+    Mockito.doAnswer(mockIterateAllUnusedSegmentsForDatasource())
+           .when(segmentsMetadataManager)
+           .iterateAllUnusedSegmentsForDatasource(
+               ArgumentMatchers.any(),
+               ArgumentMatchers.any(),
+               ArgumentMatchers.any(),
+               ArgumentMatchers.any(),
+               ArgumentMatchers.any());
```

Comment:
```suggestion
               ArgumentMatchers.any()
           );
```

Reply:
Interestingly, checkstyle doesn't like this suggested indentation. It fails with the following error:
```
MetadataResourceTest.java:140: 'method call rparen' has incorrect indentation level 11, expected level should be 6. [Indentation]
Audit done.
```
It seems like a bug in checkstyle. I have reverted this change for now.